node/silkworm/downloader/rpc/peer_min_block.hpp

```diff
@@ -14,8 +14,8 @@
    limitations under the License.
 */
 
-#ifndef SILKWORM_PEERMINBLOCK_HPP
-#define SILKWORM_PEERMINBLOCK_HPP
+#ifndef SILKWORM_PEER_MIN_BLOCK_HPP
```

Comment:
Same comment as above for include guards.
Also please add `#pragma once` (I know is pleonastic but speeds up MSVC builds)